drivers/docker/fingerprint.go

```diff
@@ -167,18 +166,7 @@ func (d *Driver) buildFingerprint() *drivers.Fingerprint {
 		fp.Attributes["driver.docker.runtimes"] = pstructs.NewStringAttribute(
 			strings.Join(runtimeNames, ","))
 		fp.Attributes["driver.docker.os_type"] = pstructs.NewStringAttribute(dockerInfo.OSType)
-
-		if runtime.GOOS == "windows" && dockerInfo.OSType == "linux" {
```

Comment:
What happens if we're running Windows 2016 where we can't run Linux LCOW workloads?

Reply:
You are right, we should leave this for old versions of Windows.